README.md

```diff
@@ -16,21 +16,21 @@ pip install git+https://github.com/individual-brain-charting/api.git#egg=ibc_api
 ```python
 import ibc_api.utils as ibc
 
-# Returns a link that takes you to EBRAINS login page and stores an access token locally
-ibc.authenticate()
-
 # Fetch info on all available files
 # Load as a pandas dataframe and save as ibc_data/available_{data_type}.csv 
 db = ibc.get_info(data_type="volume_maps")
 
 # Keep statistic maps for sub-08, for task-Discount
-filtered_db = ibc.filter_data(db, subject_list=["sub-08"], task_list=["Discount"])
+filtered_db = ibc.filter_data(db, subject_list=[8], task_list=["Discount"])
```

Comment:
Why have you changes subjects from strings to int ? I find it less logical/readable/BIDS friendly.

Reply:
Actually, pybids creates `subject` column in the dataframe with subject numbers (and session numbers, run numbers) as int. So I had to make that change to accommodate for that.

Reply:
OK. I think that this is a bid idea. Maybe leave a comment in the code to notify that.